Sources/ProjectDescription/Cache.swift

```diff
@@ -23,14 +23,17 @@ public struct Cache: Codable, Equatable {
 
     /// A list of the cache profiles.
     public let profiles: [Profile]
+    /// The path where the cache will be stored, if `nil` it will be the default location.
+    public let path: Path?
 
     /// Returns a `Cache` instance containing the given profiles.
     /// If no profile list is provided, tuist's default profile will be taken as the default.
     /// If no profile is provided in `tuist cache --profile` command, the first profile from the profiles list will be taken as the default.
     ///
     /// - Parameter profiles: Profiles to be choosen from
+    /// - Parameter path: The path where the cache will be stored, if `nil` it will be the default location
```

Comment:
nitpick: I'd change the comment the same way as above + I'd use `- Parameters: ` for documentation instead of writing twice `Parameter`